app/views/rsvps/thank_you.html.erb

```diff
@@ -1,6 +1,10 @@
 <div class="rsvp">
   <%= render 'header' %>
-  <p>Thank you for your RSVP.</p>
+  <%- if @total_count > @event.capacity %>
```

Comment:
Wondering if this should be `>=`

Reply:
I don't think so. The count includes your just created RSVP for you and your guests, so even if its at capacity for a user, that's fine. It's only for being over.

Reply:
Ah, I see